test/types/enum/vass/enum-to-uint-param-1.chpl

```diff
@@ -1,4 +1,4 @@
 
 enum e { a=-1, b=1 };
-var i:uint(8) = e.b;
+var i = e.b: uint(8);
```

Comment:
Let us also make 'i' a `param`, if that works. I think that was my original intention. Otherwise this test offers little in addition to enum-to-uint-1.chpl .